packages/blockchain-api/test/mockTokenTxs.ts

```diff
@@ -358,6 +358,7 @@ const mockTokenTxs = {
                 {
                   node: {
                     fromAddressHash: '0xf4314cb9046bece6aa54bb9533155434d0c76909',
+                    fromAccountHash: '0xf4314cb9046bece6aa54bb9533155434d0c76910', // this should go to the `account` field
```

Comment:
Maybe we could add account hashes that are different to address hashes to one of the mock transactions so we test the case where account and address are different.